teuthology/test/test_worker.py

```diff
@@ -177,13 +177,6 @@ def test_run_with_watchdog_with_reporting(self, m_sleep, m_symlink_log, m_popen)
         m_popen.return_value = m_proc
         worker.run_with_watchdog(process, config)
         m_symlink_log.assert_called_with(config["worker_log"], config["archive_path"])
-        expected_cmd = "teuthology-report -v -D -r the_name -j 1"
-        m_popen.assert_called_with(
-            expected_cmd,
-            shell=True,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.STDOUT
-        )
```

Comment:
Not sure I understand why this is removed?

Reply:
Because the command is no longer called because I removed the the special handling which does that call.